### ISSUE TITLE
wrong version returned for openssl

### ISSUE THREAD

**Issue by IlanBeloglovsky** [dvershinin/lastversion#33]
Hi used the tool (which is awesome BTW) to fetch the last version of https://github.com/openssl/openssl 
but this was the result i got:
![image](https://user-images.githubusercontent.com/75306021/104157206-5c8f8500-53f3-11eb-9ef0-318e29af041c.png)
But if i go to the repo and look at the releases i see that the last version is:
![image](https://user-images.githubusercontent.com/75306021/104157277-85b01580-53f3-11eb-8d5e-09230664b88e.png)

**Comment by dvershinin** [comment]
Hi @IlanBeloglovsky ,

To give you a quick answer, `lastversion` probably will choke on it in the near time to come as there's no easy fix.

Internally it normalizes versions to [PEP specification version format](https://www.python.org/dev/peps/pep-0440/#public-version-identifiers), which can be applied in 99% cases to non-Python version identifiers.

But the problem with openssl is how they are using actual Latin letters like 1.1.1**i** for post releases.
Instead of e.g. 1.1.1p9.

The fix for it would probably be to make sure `lastversion` has some known configuration for  OpenSSL repo,
convert to proper version format internally for sorting across releases, then reconverting the output to expected (for OpenSSL) format. Will update the ticket when this is implemented.

**Comment by dvershinin** [comment]
Things should be alright with the newly released lastversion 1.2.5 :-)

Didn't have to hardcode anything; we're now checking if the supposed version ends with `<digit><single letter>` then do the needed conversions for sorting and output.

Branch selectors should work as well, e.g.

```
# lastversion openssl:1.0
# 1.0.2u
```

```
# lastversion openssl:1.1    
# 1.1.1i
```

**Comment by IlanBeloglovsky** [comment]
Awesome! thank you very much
